### ISSUE TITLE
Error occur when expand spring-hadoop "Plugins" node

### ISSUE THREAD

**Issue by ZhengJun003** [Microsoft/vscode-maven#296]
**Describe the bug**
This issue will also appear in previous versions.

**To Reproduce**
1. Clone code from [https://github.com/spring-projects/spring-hadoop-samples.git](https://github.com/spring-projects/spring-hadoop-samples.git)
2. Open the cloned project.
3. Expand the spring-hadoop-samples node, it's a "Plugins" node
4. Expand the "Plugins" node, error occur

**Environments (please complete the following information as much as possible):**
 - OS: Windows 10/OS X
 - VS Code version: 1.32.3
 - Extension version RC#14

**Screenshots**
![image](https://user-images.githubusercontent.com/41051257/55394970-2ca32480-5573-11e9-9ba9-1fa6d3f4fda9.png)

**Additional context**
Maven for Java output error information:
```
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Child module c:\Users\testuser\Documents\spring-hadoop-samples\parent\mapreduce of c:\Users\testuser\Documents\spring-hadoop-samples\parent\pom.xml does not exist @ 
[ERROR] Child module c:\Users\testuser\Documents\spring-hadoop-samples\parent\hive of c:\Users\testuser\Documents\spring-hadoop-samples\parent\pom.xml does not exist @ 
[ERROR] Child module c:\Users\testuser\Documents\spring-hadoop-samples\parent\pig of c:\Users\testuser\Documents\spring-hadoop-samples\parent\pom.xml does not exist @ 
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.springframework.samples:spring-hadoop-samples:1.0.0.BUILD-SNAPSHOT (c:\Users\testuser\Documents\spring-hadoop-samples\parent\pom.xml) has 3 errors
[ERROR]     Child module c:\Users\testuser\Documents\spring-hadoop-samples\parent\mapreduce of c:\Users\testuser\Documents\spring-hadoop-samples\parent\pom.xml does not exist
[ERROR]     Child module c:\Users\testuser\Documents\spring-hadoop-samples\parent\hive of c:\Users\testuser\Documents\spring-hadoop-samples\parent\pom.xml does not exist
[ERROR]     Child module c:\Users\testuser\Documents\spring-hadoop-samples\parent\pig of c:\Users\testuser\Documents\spring-hadoop-samples\parent\pom.xml does not exist
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

**Comment by Eskibear** [comment]
It looks when failing to create effective-pom, it keeps re-trying for ever.

**Comment by Eskibear** [comment]
the fix has been shipped with v0.16.1, please have a try. Now it should only leaves one error message and stop retrying.

**Comment by ZhengJun003** [comment]
Verified in v0.16.1 on Win&Mac platform, this issue is fixed.
